plugins/wacom-usb/meson.build

```diff
@@ -34,7 +36,10 @@ if get_option('tests')
   env.set('G_TEST_BUILDDIR', meson.current_build_dir())
   e = executable(
     'wacom-usb-self-test',
-    rustgen.process('fu-wac.rs'),
+    rustgen.process(
+      'fu-wac.rs',
+      'fu-wac-module-bluetooth-id9.rs',
```

Comment:
I think just use `fu-wac.rs` please

Reply:
To be clear: `fu-wac-module-bluetooth-id9.rs` should be removed from this instance of `rustgen.process()` (for the `wacom-usb-self-test` executable) but should remain in use for the previous instance of `rustgen.process()` (for the `fu_plugin_wac` library). Correct?

Reply:
Ack